packages/common/core-interfaces/src/erasedType.ts

```diff
@@ -0,0 +1,81 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * "Erased" handle which can be used to expose a opaque/erased version of a type without referencing the actual type.
```

Comment:
```suggestion
 * "Erased" reference which can be used to expose a opaque/erased version of a type without referencing the actual type.
```

Reply:
I was going for https://en.wikipedia.org/wiki/Handle_(computing). What do you think of:
```suggestion
 * "Erased" [handle](https://en.wikipedia.org/wiki/Handle_(computing)) which can be used to expose a opaque/erased version of a type without referencing the actual type.
```